cmd/notify-http.go

```
@@ -59,16 +84,30 @@ func (n httpNotify) Fire(entry *logrus.Entry) error {
 	fmt.Println("We got a HTTP event fired.")
 	fmt.Println(entryStr)
 
-	buf := bytes.NewBufferString(entryStr)
-	response, err := http.Post(n.Addr, "application/json", buf)
+	httpPostBody1 := httpPostBody{
+		Records: entry.Data["Records"],
+		Key:     entry.Data["Key"].(string),
+	}
+
+	itemsStr, err := json.Marshal(httpPostBody1)
+	if err != nil {
+		entry.Warn("Cannot convert request body to JSON.")
```

We should return an error here and not proceed. If it proceeds it will write an empty buffer.

---

👍 - OK I wasn't 100% how you were handling warnings. I've pushed an update.